compass/ocean/tests/isomip_plus/initial_state.py

```diff
@@ -125,12 +129,20 @@ def _compute_initial_condition(self):
 
         ds['landIceFraction'] = xr.where(mask, ds.landIceFraction, 0.)
 
-        ref_density = constants['SHR_CONST_RHOSW']
-        landIcePressure, landIceDraft = compute_land_ice_pressure_and_draft(
-            ssh=ds.ssh, modify_mask=ds.ssh < 0., ref_density=ref_density)
+        land_ice_draft = ds.ssh
+        if thin_film_present:
+            land_ice_thickness = ds.landIceThickness
+            land_ice_density = compute_land_ice_density_from_draft(
+                land_ice_draft, land_ice_thickness,
+                floating_mask=ds.landIceFloatingMask)
```

Comment:
I think we should just include this in the config options, like I am doing for Polaris:
https://github.com/xylar/polaris/blob/add-isomip-plus-init/polaris/ocean/tasks/isomip_plus/isomip_plus.cfg#L27
We know what BISICLES used in their simulation, as it's also the value prescribed in the paper (Table 1 in [Asay-Davis et al. 2016](https://gmd.copernicus.org/articles/9/2471/2016/gmd-9-2471-2016.html)).